schemas/context/destination-segment.example.1.json

```diff
@@ -0,0 +1,6 @@
+{
+  "xdm:destinationID": "1234",
+  "xdm:segmentID": "8769",
```

Comment:
segmentID is a string, and not numeric. Example of a Segment ID: `0b12f30c-2260-442e-9120-2b12e96c0c40`